aptos-move/e2e-testsuite/Cargo.toml

```diff
@@ -19,7 +19,7 @@ aptos-crypto = { workspace = true }
 aptos-framework = { workspace = true }
 aptos-gas-algebra = { workspace = true }
 aptos-gas-meter = { workspace = true }
-aptos-gas-schedule = { workspace = true }
+aptos-gas-schedule = { workspace = true, features = ["testing"] }
```

Comment:
why is this?

Reply:
This is another bug caused by the original PR. If you run `cargo test` for the e2e-testsuite crate alone, you'll notice half of the test failing due to error GAS_UNIT_PRICE_BELOW_MIN_BOUND. CI failed to catch this because it performs feature unification for all crates.